tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/Program.cs

```diff
@@ -88,7 +88,10 @@ static async Task HandleGenerateCodeFile(IEnumerable<string> swaggers, string ou
         {
             if (!File.Exists(swaggerFilePath))
             {
-                throw new FileNotFoundException(swaggerFilePath);
+                // Suppressing this error when swagger file is missing or path is incorrect
+                // Ideally any incorrect swagger file path reference in README should be caught by validation tool
+                Console.WriteLine($"Invalid file path to swagger file. Skipping {swaggerFilePath}.");
```

Comment:
You sure this does not pose another issue?

Reply:
My assumption is that we shouldn't have reached here in most cases when readme is expected to have valid swagger file paths. Still wondering how previous change got past the pipeline checks. Only possibility is that validation tool (if any) validated files in the first section with a given tag and ignored duplicate tag.

@ruowan Don't we have any validation in swagger pipeline to ensure file paths are correct in readme when a PR is submitted? We have an issue of baseline swagger API review gen failing because readme on target branch has incorrect file path to swagger files.

@weshaggard FYI....

Reply:
I don't have enough context but I do think we should try and gracefully handle any cases where the swagger file isn't present. We shouldn't crash.

Reply:
We do have Avocado validation in spec PR pipeline to validate file path in readme is correct.

Reply:
Wondering why the Avocado pipeline didn't fail in this case. Should we open an issue to follow up?